programs/library-bridge/CatBoostLibraryAPI.h

```diff
@@ -0,0 +1,49 @@
+#pragma once
```

Comment:
The code in this file is taken from https://clickhouse.com/codebrowser/ClickHouse/src/Interpreters/CatBoostModel.cpp.html#DB::CatBoostWrapperAPI